File: Selenium/Project/Activity7.py
from selenium import webdriver
from selenium.webdriver.common.by import By

# Create an instance of the Firefox webdriver
driver = webdriver.Firefox()

# Open the webpage
driver.get("https://alchemy.hguy.co/lms")

# Click on the "All Courses" link
driver.find_element(By.XPATH, "//a[contains(text(),'All Courses')]").click()

# Find all elements with class name "col-sm-8"
elements = driver.find_elements(By.CLASS_NAME, "col-sm-8")

# Get the count of elements
count = len(elements)

# Print the number of courses
print("Number of courses:", count)

# Close the browser
driver.quit()

File: Selenium/Project/Activity7.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;
import org.testng.annotations.Test;

import java.util.List;

public class Activity7 extends BaseClass{
    @Test
    public void activity7() {
        int c = 0;
        driver.findElement(By.xpath("//a[contains(text(),'All Courses')]")).click();
        List<WebElement> list = driver.findElements(By.className("col-sm-8"));
        for (WebElement count : list) {
            c = list.size();
        }
        System.out.println("Number of courses : " + c);
    }
}
